dolphin/db/sqlalchemy/api.py

```diff
@@ -298,17 +298,42 @@ def storage_get_all(context, marker=None, limit=None, sort_keys=None,
     return query.all()
 
 
+def storage_delete(context, storage_id):
```

Comment:
shoul we not need session here?

Reply:
Means we need to use session here?

Reply:
model_query() would create session by itself

Reply:
ok